proto/zipkin.proto

```diff
@@ -0,0 +1,233 @@
+//
```

Comment:
please indicate the origin (url) of this file

Reply:
Done. We didn't add URL for [swagger](https://github.com/jaegertracing/jaeger-idl/blob/master/swagger/zipkin2-api.yaml) and [thrift](https://github.com/jaegertracing/jaeger-idl/blob/master/thrift/zipkincore.thrift), should we add for them too?